db/migrate/20170526114520_create_posts.rb

```diff
@@ -2,8 +2,9 @@ class CreatePosts < ActiveRecord::Migration[5.1]
   def change
     create_table :posts do |t|
       t.string :message
-
+      t.belongs_to :user, index: true
```

Comment:
the schema specifies created_at and updated_at columns: but it's not included in the migration file. Is that an issue?

Reply:
Or maybe it's included in an older migration file...

Reply:
Nope it has never been there. I think it's related to t.timestamps on line 6 but don't know how this works actually

Reply:
I agree with Mathilde, the `t.timestamps` in the migration must tell rails to create two generic 'created_at'/'updated_at' columns with datetime data types